adoc/chapters/programming_interface.adoc

```diff
@@ -2258,9 +2287,12 @@ info::device::atomic_fence_order_capabilities
 ----
 
     @ [.code]#std::vector<memory_order>#
-   a@ Returns the set of memory orderings supported by [code]#atomic_fence# on the device,
-      which is guaranteed to include [code]#relaxed#, [code]#acquire#,
-      [code]#release# and [code]#acq_rel#.
+   a@ Returns the set of memory orders supported by [code]#atomic_fence# on
+      the device.  When a device returns a "stronger" memory order in this set,
+      it must also return all "weaker" memory orders.  At a minimum, each
+      device must support [code]#memory_order::relaxed#,
+      [code]#memory_order::acquire#, [code]#memory_order::release#, and
+      [code]#memory_order::acq_rel#.
```

Comment:
Is it true?

Reply:
I think this is the same question you are asking above, so we can discuss it there.